### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by LogA1Cyber** [3kh0/ext-remover#106] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My school just made it so that the hacks don't work

**Comment by synapt0** [comment, marked as answer]
If you update your Chromebook it doesn't work anymore.

**Reply by LogA1Cyber** [reply]
my chomrebook did not update, they did something else, I looked at chrome://policy (I found this online)
and it has
javascriptblockedforurls javascript://*
javascriptblockedforsites javascript://*

**Reply by LogA1Cyber** [reply]
I don't know if that is what is breaking it, so I am guessing

**Reply by synapt0** [reply]
then ur fcked
